charts/kubescape-cloud-operator/templates/kubevuln/deployment.yaml

```diff
@@ -44,6 +44,9 @@ spec:
       imagePullSecrets:
       - name: {{ toYaml .Values.imagePullSecrets }}
       {{- end }}
+      securityContext:
```

Comment:
Was this committed intentionally?

Reply:
yes, it is needed to fix the pv permissions when we mount it